src/odemis/driver/test/xt_client_test.py

```diff
@@ -245,16 +249,60 @@ def _compute_expected_duration(self):
 
     def test_acquire(self):
         """Test acquiring an image using the Detector."""
+        init_dwell_time = self.scanner.dwellTime.value
         self.scanner.dwellTime.value = 25e-9  # s
         expected_duration = self._compute_expected_duration()
-
         start = time.time()
         im = self.detector.data.get()
         duration = time.time() - start
-
         self.assertEqual(im.shape, self.scanner.resolution.value[::-1])
         self.assertGreaterEqual(duration, expected_duration, "Error execution took %f s, less than exposure time %d." % (duration, expected_duration))
         self.assertIn(model.MD_DWELL_TIME, im.metadata)
+        # Set back dwell time to initial value
+        self.scanner.dwellTime.value = init_dwell_time
+
+    def test_stop_acquisition(self):
+        """Test stopping the acquisition of an image using the Detector."""
+        init_dwell_time = self.scanner.dwellTime.value
+        init_resolution = self.scanner.resolution.value
+        self.scanner.dwellTime.value = 25e-9  # s
+        # Set resolution to a high value for a long acquisition that will be stopped.
+        self.scanner.resolution.value = (3072, 2048)  # px
+        self.detector.data.subscribe(self.receive_data)
+        self.detector.data.unsubscribe(self.receive_data)
+        # Set resolution to a low value for a quick acquisition.
+        self.scanner.resolution.value = (768, 512)  # px
+        im = self.detector.data.get()
+        # Check that the acquired image has the last set resolution.
+        self.assertEqual(im.shape, self.scanner.resolution.value[::-1])
+        self.assertIn(model.MD_DWELL_TIME, im.metadata)
+        # Set back resolution and dwell time to initial values
+        self.scanner.dwellTime.value = init_dwell_time
+        self.scanner.resolution.value = init_resolution
+
+    def test_live_chance(self):
```

Comment:
typo? chance -> change

Reply:
It's to indicate there's only a small chance this test passes :grinning:. Just kidding I'll update it.